tests/js/spec/views/dashboardsV2/gridLayout/detail.spec.jsx

```diff
@@ -641,14 +647,22 @@ describe('Dashboards > Detail', function () {
       await tick();
       wrapper.update();
 
-      // Enter Add Issue Widget mode
       wrapper
-        .find('Controls Button[data-test-id="dashboard-add-issues-widget"]')
+        .find('Controls Button[data-test-id="add-widget-library"]')
         .simulate('click');
 
-      const modal = await mountGlobalModal();
+      let modal = await mountGlobalModal();
       await tick();
-      await modal.update();
+      await tick();
+      modal.update();
+      modal.find('CustomButton').simulate('click');
+      modal = await mountGlobalModal();
+      await tick();
+      modal.update();
+      modal.find('SelectControl').first().props().onChange({value: 'table'});
+      await tick();
+      modal.update();
+      modal.find('RadioGroup').props().onChange('issue');
```

Comment:
Should try to avoid the async mount modal stuff so we don't run into timeouts on acceptance tests that we may have to fix down the road?

Reply:
Good call, I actually removed this test from the non grid details.spec.jsx since it's not longer relavent (this add issue button is now gone). Going to remove this test as well!